docroot/themes/contrib/civictheme/config/install/core.entity_form_display.node.civictheme_page.default.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
     - field.field.node.civictheme_page.field_c_n_topics
     - node.type.civictheme_page
   module:
+    - content_moderation
```

Comment:
the config linter was not running and some of the configs became stale - updating them here